src/analytics/useDpal.ts

```diff
@@ -0,0 +1,32 @@
+import { useFlag } from '@unleash/proxy-client-react';
+import { useEffect, useRef } from 'react';
+
+function useDPAL() {
+  const dpalAdded = useRef(false);
+  const enableDPAL = useFlag('platform.chrome.analytics.dpal');
```

Comment:
I'll be sure to give our itless team a heads up _not_ to enable this flag when they sync all the flags.